### PR TITLE
docs(apim): initialize API import documentation

### PR DESCRIPTION
**Pull request by marcambier**
https://github.com/gravitee-io/issues/issues/7003
https://github.com/gravitee-io/issues/issues/7013

### CODE REVIEW

File: pages/apim/3.x/user-guide/publisher/import-apis.adoc [hunk 1]
```diff
@@ -0,0 +1,67 @@
+= Import APIs
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_publisherguide_import_apis.html
+:page-folder: apim/user-guide/publisher
+:page-layout: apim3x
+:page-liquid:
+
+== Overview
+
+The following sections describe the process of importing an API from his JSON definition.
```
Comment (a-cordier) [inline pages/apim/3.x/user-guide/publisher/import-apis.adoc:10]:
```suggestion
The following sections describe the process of importing an API from its JSON definition.
```

File: pages/apim/3.x/user-guide/publisher/import-apis.adoc [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+Use the **POST /api/import** endpoint, on the targeted environment (link:https://gravitee-docs.gravitee.io/apim/3.x/management-api/3.15/#operation/importApiDefinition[API reference^]).
+
+NOTE: It will raise an error if there is already an existing API on the targeted environment with the same **crossId** as your API definition.
+
+=== Updating an existing API from an API definition
+
+Use the **PUT /api/{apiId)/import** endpoint, on the targeted environment (link:https://gravitee-docs.gravitee.io/apim/3.x/management-api/3.15/#operation/updateWithDefinitionPUT[API reference^]).
+
+In the URL path parameter, user the API technical **ID** that can be found in the console's URL, or in the import endpoint HTTP response.
```
Comment (a-cordier) [inline pages/apim/3.x/user-guide/publisher/import-apis.adoc:31]:
```suggestion
In the URL path parameter, use the API technical **ID** that can be found in the console's URL, or in the import endpoint HTTP response.
```

File: pages/apim/3.x/user-guide/publisher/import-apis.adoc [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+=== How to get an API definition ?
+
+You can get the definition of your API, by exporting it from the APIM console.
+Or, by using the export endpoint
+`**GET /apis/{api.id}/export**`
+(link:/apim/3.x/management-api/3.16/#operation/exportApiDefinition[API reference^]).
+
+=== What does it contain ?
+
+In this definition, each entity (API, plan, or page) has :
```
Comment (a-cordier) [inline pages/apim/3.x/user-guide/publisher/import-apis.adoc:27]: Maybe we should state more accurately what entity contains a cross ID or not (e.g. group, members, roles do not) ?

**Comment by mouligno** [comment]
giphy (5)

**Pull request merged**